feature-libs/organization/administration/components/budget/budget.config.ts

```diff
@@ -80,10 +80,57 @@ export const budgetCmsConfig: CmsConfig = {
   },
 };
 
-export function budgetTableConfigFactory(): TableConfig {
-  return budgetTableConfig;
+// TODO: (CXSPA-7155) - Remove feature flag and legacy config next major release
+export function budgetTableConfigFactory(
+  featureConfigService?: FeatureConfigService
+): TableConfig {
+  if (featureConfigService?.isEnabled('a11yOrganizationLinkableCells')) {
```

Comment:
While I like the reference to `legacyConfig` and I think it will be easier to clean up, I think we should retain the original names for the legacy configs in case they have been exported and used elsewhere. We should use something like `newConfig` instead to reference our modern changes and change the reference back to `regularConfig` once we can clean up our breaking changes.

Reply:
Please do this for the remaining configs too ;)